kornia/contrib/extract_patches.py

```diff
@@ -19,7 +19,7 @@ def compute_padding(
         window_size: the size of the sliding window used while extracting patches.
 
     Return:
-        The required padding for `(top, bottom, left, right)` as a tuple of 4 ints.
+        The required padding for `(vertical, horizontal)` as a tuple of 2 ints.
```

Comment:
why we shouldn't support `(top, bottom, left, right)` anymore?

Reply:
This is due to `fold` only accepting padding as a tuple of 2 ints.

Reply:
i see -- then i think is better to make it as a v2 and add a deprecation warning that the behavior will be changed in the future (after 0.7.2) cc @edgarriba

Reply:
something like:

```python
from kornia.utils.helpers import deprecated

@deprecated('compute_padding_v2', '0.7.0', 'This function behaviour will be replaced by compute_padding_v2 after kornia-0.7.2.')
def compute_padding
...

def compute_padding_v2 # with your implementation

Reply:
agree

Reply:
What about `CombineTensorPatches`? Originally it did not take stride as a parameter. Does it need a deprecation note as well?

Also I just want to point out that the main use of `compute_padding` is to compute the padding required to do the Extract - Combine process correctly. So the change shouldn't affect users if they have used it like so.

```python
import torch

from kornia.contrib import CombineTensorPatches, ExtractTensorPatches, compute_padding

# Original size
h, w = 4, 4
# Window size
win = 3
# Stride
stride = 1
# Padding size
padding = compute_padding(original_size=(h,w), window_size=win)

image = torch.randn(2, 3, h, w)
tiler = ExtractTensorPatches(window_size=win, stride=stride, padding=padding)
merger = CombineTensorPatches(original_size=(h,w), window_size=win, stride=stride, unpadding=padding)
image_tiles = tiler(image)
restored_image = merger(image_tiles)
assert torch.allclose(image,restored_image)
```

Reply:
right, it is very unlikely that users are using compute_padding directly out of this setup, let's just go ahead with it